core/trino-spi/src/main/java/io/trino/spi/eventlistener/ColumnInfo.java

```diff
@@ -46,4 +47,19 @@ public Optional<String> getMask()
     {
         return mask;
     }
+
+    @Override
```

Comment:
Should we migrate this class to record?

Reply:
This is an SPI class, although marked as `@Unstable`, so we can migrate it to record, but still we should do it carefully